### ISSUE TITLE
Demolition truck- explosion on ore removes ore

### ISSUE THREAD

**Issue by sorcerer86pt** [OpenRA/ra2#560]
you can see movie of explosion on ore here:

https://mega.nz/#!sEpD0KKa!RaAafUEi9iAtQVb34B5cTyLAnuLmHzinsDzX0-hTQqE

**Comment by Mailaender** [comment]
and it shouldn't?

**Comment by sorcerer86pt** [comment]
No, due to the size of some maps. Although resource starvation is a mechanic of some RTS, it is not on C&C series. In fact, destruction of resources only appeared after, in tiberium wars, with the gdi Sonic weapons. It didn't comeback in future expansions, nor did this mechanic surfaced in red alert 3. Also it's more of a big that balancing, since it's the explosion tike being superimposed on resource tile

**Comment by Mailaender** [comment]
I think this behavior was inherited from the Red Alert demo truck and nuke.

**Comment by sorcerer86pt** [comment]
It's a very real possibility. The issue here is , if compared to OG RA2, we mantain the original behavior and no ore destruction, or mantain the logic of RA1 demo truck/nuke. Remember that we have small maps, for 1vs1 with some low patches of ore, and overall game theme.

**Comment by GraionDilach** [comment]
>Although resource starvation is a mechanic of some RTS, it is not on C&C series.

Tiberian Sun and Red Alert 2 would like to have a word with you.
https://www.modenc.renegadeprojects.com/Tiberium_(INI_flag)

TLDR:
All resources in TS/RA2 have wood armor and thereby destructible by warheads with the `Wood=yes` flag on a one-by-one stage basis.
The `WallAbsoluteDestroyer=yes` warhead flag introduced in YR was able to delete all stages of a resource, but it was only used on tank shells and wasn't used such drastically as the OP.
These flags acknowledge `CellSpread` though.

**Comment by sorcerer86pt** [comment]
So, not not a big.
